src/sqlfluff/rules/L057.py

```diff
@@ -53,6 +54,7 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
         self.quoted_identifiers_policy: str
         self.unquoted_identifiers_policy: str
         self.allow_space_in_identifier: bool
+        self.additional_allowed_identifiers: str
```

Comment:
These aren't really "identifiers". Rather, they are additional _characters_ allowed in identifiers. Perhaps call it `additional_allowed_identifier_characters` for clarity?

Reply:
I've gone with `additional_allowed_characters` as it's a rule-specific config specific to this rule, so think it's obvious it's for identifiers.